### PR TITLE
codecov: ignore build scripts in coverage and don't comment on PRs

### PR DESCRIPTION
**Pull request by hmaarrfk**
Ignore known build scripts from codecov. This also supresses codecov from commenting on the PR, but the check at the bottom should remain.

https://docs.codecov.io/docs/ignoring-paths

Curious to see how much we are actually testing the code.

## For reviewers
- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.
- [ ] Consider backporting the PR with `@meeseeksdev backport to v0.14.x`

### CODE REVIEW

**Comment by hmaarrfk** [comment]
@scikit-image/core how do you feel about this PR. As much as I would love to have github mark codecov as spam by default, it seems that this is the more direct fix.

File: .codecov.yml [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+  - "scripts/skivi.py"
+  - "setup.py"
+  - "**/setup.py"
+
+coverage:
+  status:
+    project:
+      default:
+        # Allow the coverage to drop by 0.1% even when posting success
+        # Drops on the order 0.01% are typical even when no change occurs
```
Comment (soupault) [inline .codecov.yml]: Doesn't match still :smile: .

Comment (jni) [reply .codecov.yml]: @soupault yes it does. He's saying codecov can report 0.01% drop even when nothing has changed, so we set the tolerance to 0.1%.

Comment (hmaarrfk) [reply .codecov.yml]: @soupault not sure if the rewording helped. 0.1 and 0.01 look like typos ;)

**Pull request merged**